docs/framework/winforms/controls/how-to-add-web-browser-capabilities-to-a-windows-forms-application.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: Add web browser capabilities to app
+title: Add web browser capabilities to a Windows Forms application
```

Comment:
this makes the title a bit repetitive 
![image](https://user-images.githubusercontent.com/12971179/73094078-bde1df80-3e94-11ea-9e5a-ec31812ef942.png)

I wouldn't mind adding the article there though

Reply:
@mairaw, Oh. I completely forgot about title suffix.

Reply:
Good case for my "let's get rid of the global suffix" 👾 😁